packages/snowpack/src/commands/build.ts

```diff
@@ -70,41 +93,22 @@ export async function command(commandOptions: CommandOptions) {
   mkdirp.sync(buildDirectoryLoc);
   mkdirp.sync(internalFilesBuildLoc);
 
-  console.log = (...args) => {
-    messageBus.emit('CONSOLE', {level: 'log', args});
-  };
-  console.warn = (...args) => {
-    messageBus.emit('CONSOLE', {level: 'warn', args});
-  };
-  console.error = (...args) => {
-    messageBus.emit('CONSOLE', {level: 'error', args});
-  };
   let relDest = path.relative(cwd, config.devOptions.out);
   if (!relDest.startsWith(`..${path.sep}`)) {
     relDest = `.${path.sep}` + relDest;
   }
-  paint(
-    messageBus,
-    config.plugins.map((p) => p.name),
-    {dest: relDest},
-    undefined,
-  );
 
   for (const runPlugin of config.plugins) {
     if (runPlugin.run) {
-      messageBus.emit('WORKER_START', {id: runPlugin.name});
       runPlugin
         .run({
           isDev: false,
-          log: (msg, data) => {
+          log: (msg, data = {}) => {
```

Comment:
nice!

Reply:
tbh, I'm not sure if I realized that `log()` required a special `msg` keyword (`WORKER_START`, etc). I thought that it could just log any string message to the console, but I guess the `run-script` needs this to update the dashboard. 

I think this forces a question: do we make this function publicly documented? Or, do we make it private, and then publicly document a wrapper around it that specifically just logs things to the console (no special keyword required). 

Happy to talk through in person if that doesn't make sense.

Reply:
It’s already public because we expose it to all the plugins—it’s the same interface that dev, build, and install all use. +1 on thinking it through and documenting it a little better (I think that’s what #523 was trying to accomplish)

Reply:
I think you suggested this, but I’d be +1 in favor of adopting Rollup’s logging strategy with `this.warn(…)` / `this.error(…)` etc.

Reply:
in that case lets remove `log()` from the documented plugin interface for 2.7.0. Making the current `log()` function public/documented would effectively make our `messageBus` interface public, which it definitely isn't ready for

Reply:
For the plugins that still need it (run-script, build-script) let's expose it as an undocumented/private `_logMessage` argument.

Reply:
Update: removed from docs in #627